docs/products/opensearch/howto/migrating_elasticsearch_data_to_aiven.rst

```diff
@@ -0,0 +1,82 @@
+Migrate Elasticsearch data to Aiven for OpenSearch
+==================================================
+
+We recommend migrating to Aiven for OpenSearch by reindexing from
+your remote cluster. The same process works for migrating from Aiven for
+OpenSearch to a self-hosted Elasticsearch service.
+
+For a larger number of indexes, we recommend that you create a script to
+run these steps automatically.
+
+As Aiven for OpenSearch does not support joining external
+Elasticsearch servers to the same cluster, online migration is not
+currently possible.
+
+.. important::
+    Migrating from Elasticsearch to OpenSearch may affect the connectivity between client applications and your service. For example, some code included in clients or tools may check the service version, which might not work with OpenSearch. We recommend that you check the following OpenSearch resources for more information:
+
+    * `OpenSearch release notes <https://github.com/opensearch-project/OpenSearch/blob/main/release-notes/opensearch.release-notes-1.0.0.md>`_
+    * `OpenSearch Dashboards release notes <https://github.com/opensearch-project/OpenSearch-Dashboards/blob/main/release-notes/opensearch-dashboards.release-notes-1.0.0.md>`_
+    * `Frequently asked questions about OpenSearch <https://opensearch.org/faq/>`_
+
+
+To migrate or copy data:
+
+#. Create a hosted Aiven for OpenSearch service.
```

Comment:
We could link to our 'create a service' instructions when live.

Reply:
Yep, I was thinking the same :)